### PR TITLE
dependencies for gentoo

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
see https://github.com/ros/rosdistro/issues/15082

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
Fixed in the next commit

**Comment by Alessandro-Barbieri** [comment]
finished

**Comment by Alessandro-Barbieri** [comment]
I think this is safe to merge.

File: rosdep/base.yaml [hunk 1]
```diff
@@ -365,7 +369,7 @@ can-utils:
 catkin_pkg:
   gentoo:
     portage:
-      packages: [dev-python/catkin_pkg]
+      packages: [dev-util/catkin_pkg]
```
Comment (allenh1) [inline rosdep/base.yaml]: Please remove this change.

File: rosdep/base.yaml [hunk 2]
```diff
@@ -4062,46 +4272,56 @@ tcsh:
...
+  gentoo: [dev-texlive/texlive-fontsrecommended]
   macports: [texlive-fonts-recommended]
   ubuntu: [texlive-fonts-recommended]
 texlive-full:
+  gentoo: [app-text/texlive]
   ubuntu: [texlive-full]
 texlive-latex-base:
   arch: [texlive-core]
-  debian: [texlive-latex-base]
+  debian: [texlive-core]
```
Comment (allenh1) [inline rosdep/base.yaml]: Did you mean to update the debian target?

**Comment by allenh1** [comment]
Other than those two items, this looks really good. Thank you very much!

File: rosdep/base.yaml (hunk 2 above)
Comment (Alessandro-Barbieri) [inline rosdep/base.yaml]: oops

**Comment by allenh1** [comment]
Lgtm

Thank you very much!

**Review by allenh1 — APPROVED** [review]

**Review by tfoote — APPROVED** [review]

**Pull request merged**